skypeweb/skypeweb_login.c

```diff
@@ -112,15 +112,17 @@ skypeweb_login_got_pie(PurpleUtilFetchUrlData *url_data, gpointer user_data, con
 	g_string_append_printf(postdata, "js_time=%" G_GINT64_FORMAT "&", skypeweb_get_js_time());
 	g_string_append(postdata, "client_id=578134&");
 	g_string_append(postdata, "redirect_uri=https://web.skype.com/");
+
+	if (postdata->len > INT_MAX) abort();

```

Comment:
`abort()` isn't great for libpurple things.  Can you change this into a `purple_connection_error(...)` instead?